assets/dev/js/editor/elements/views/base.js

```diff
@@ -247,6 +247,9 @@ BaseElementView = BaseContainer.extend( {
 		_.defer( () => {
 			// Init container. Defer - in order to init the container after the element is fully initialized, and properties like `_parent` are available.
 			this.getContainer();
+
+			// Since the global colors were not ready, the styles are rendered again for the gradiant background color control.
+			this.renderStyles();
```

Comment:
Maybe we should check why this is happening instead of putting a bandage on it making UI glitches?

Reply:
This solution is good enough. The global colors are available after the code in this class runs.

Reply:
The element is not rendered twice only once.